<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUiMessageBox</class>
 <widget class="QWidget" name="CUiMessageBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>355</width>
    <height>68</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CUiMessageBox</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="m_info">
     <property name="font">
      <font>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string>dfasdfsdf</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="margin">
      <number>4</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="m_infoHolder" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0,0,0,0,0,1">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_2" native="true"/>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnYes">
        <property name="text">
         <string>Yes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnNo">
        <property name="text">
         <string>No</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnOk">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnDiscard">
        <property name="text">
         <string>Discard</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_btnCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
